src/msg/msg_types.h

```diff
@@ -442,16 +442,19 @@ struct entity_inst_t {
     return i;
   }
 
-  void encode(bufferlist& bl) const {
+  void encode(bufferlist& bl, uint64_t features) const {
     ::encode(name, bl);
-    ::encode(addr, bl);
+    ::encode(addr, bl, features);
```

Comment:
Doesn't this create a backwards compatibility issue?  It isn't part of an ENCODE_START/ENCODE_FINISH that includes total length and versioning.